.gitignore

```diff
@@ -1,3 +1,5 @@
+.envrc
+.venv
```

Comment:
Also curious: are you using poetry with virtualenvs.in-project set to True, and if so, why?